test/rest-api/body.test.ts

```diff
@@ -90,3 +90,49 @@ test('handles a POST request with a JSON body and "Content-Type: application/jso
 
   return runtime.cleanup()
 })
+
+test('handles a POST request with a multipart body and "Content-Type: multipart/form-data" header', async () => {
+  const runtime = await createRuntime()
+  // WORKAROUND: `FormData` is not available in `puppeteer.page.evaluate`
+  const body = `
+------WebKitFormBoundaryvZ1cVXWyK0ilQdab\r
+Content-Disposition: form-data; name="file"; filename="file1.txt"\r
+Content-Type: application/octet-stream\r
+\r
+file content\r
+------WebKitFormBoundaryvZ1cVXWyK0ilQdab\r
+Content-Disposition: form-data; name="text"\r
+\r
+text content\r
+------WebKitFormBoundaryvZ1cVXWyK0ilQdab\r
+Content-Disposition: form-data; name="text2"\r
+\r
+another text content\r
+------WebKitFormBoundaryvZ1cVXWyK0ilQdab\r
+Content-Disposition: form-data; name="text2"\r
+\r
+another text content 2\r
+------WebKitFormBoundaryvZ1cVXWyK0ilQdab--\r\n`.trimStart()
```

Comment:
Prefer using `\` (backward slash) to escape an extra new line of a formatted template literal string:

```js
const string = `\
yes
`
```